test/buffer/worker/expected-etcd-state/_files_imported_001.txt

```diff
@@ -34,14 +34,6 @@ file/imported/%%TEST_KBC_PROJECT_ID%%/my-receiver/my-export-1/%s
   "closingAt": "%s",
   "importingAt": "%s",
   "importedAt": "%s",
-  "statistics": {
-    "lastRecordAt": "%s",
-    "recordsCount": 10,
-    "recordsSize": "%s",
-    "bodySize": "%s",
-    "fileSize": "%s",
-    "fileGZipSize": "%s"
-  },
```

Comment:
Vo worker E2E testoch je `/stats` prefix ignorovany.
https://github.com/keboola/keboola-as-code/blob/95b4c5019f505c09d87c467712f2e6bb081d373d/test/buffer/worker/utils_test.go#L63-L64

Preto je tu iba odstranenie statistik zo `slice/file`, ale uz tu nie su nove kluce.
Upravim to v samostatnom PR, aby som tento PR uz viac nenafukoval.

Reply:
PR: https://github.com/keboola/keboola-as-code/pull/1380